react/features/screen-share/functions.js

```diff
@@ -23,3 +33,25 @@ export function isScreenAudioShared(state: Object) {
 export function isScreenAudioSupported() {
     return browser.isChrome() || (browser.isElectron() && isWindows());
```

Comment:
Is this supported on mobile browsers ?

Reply:
Not as far as i know.

Reply:
Ow I see, there is no distinction between desktop chrome and mobile. Will fix.

Reply:
Fixed.